app/displayers.py

```diff
@@ -0,0 +1,18 @@
+from abc import ABC, abstractmethod
+from .models import Book
```

Comment:
`.modules` is a custom module, while `abc` is an internal module. Therefore, `.modules` should be separated from the internal module imports.